dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -55,6 +56,8 @@
 from dftimewolf.cli.recipes import local_plaso
 from dftimewolf.cli.recipes import plaso_file_timesketch
 
+signal.signal(signal.SIGINT, dftw_utils.signal_handler)
```

Comment:
So many signal.. :)

To get less signal, maybe do:
```
from signal import signal
```

Reply:
Sounds good. Done.